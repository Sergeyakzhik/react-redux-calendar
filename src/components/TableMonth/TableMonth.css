.table-month td {
  height: 7rem;
  border: 1px solid black;
  text-align: left;
  position: relative;
}

.table-month th:first-child {
  color: #ea0000;
}

.table-month tr td:first-child {
  color: #ea0000;
}

.table-month td h4 {
  margin-top: 0rem;
  margin-bottom: 5rem;
  margin-left: 0.2rem;
}

.table-month {
  border-spacing: 0.08rem;
}
