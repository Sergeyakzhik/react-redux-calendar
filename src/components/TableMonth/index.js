import React from 'react';
import './style.css';

const TableMonth = props => (
  <div>
    <h2>{props.date}</h2>
    <table className="table-month">
      <thead>
        {props.tableHeader}
      </thead>
      <tbody>
        {props.tableRows}
      </tbody>
    </table>
  </div>
);

export default TableMonth;
