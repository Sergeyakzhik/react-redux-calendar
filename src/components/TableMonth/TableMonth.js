import React from 'react';
import './TableMonth.css';

let TableMonth = props => (
  <div>
    <h2>{props.date}</h2>
    <table className='table-month'>
      <thead>
        {props.tableHeader}
      </thead>
      <tbody>
        {props.tableRows}
      </tbody>
    </table>
  </div>
);

export default TableMonth;
