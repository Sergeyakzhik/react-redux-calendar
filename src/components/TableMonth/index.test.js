import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

import TableMonth from './index';

Enzyme.configure({ adapter: new Adapter() })

describe('TableMonth', () => {
  let props = {
    tableRows: [],
    tableHeader: [],
    date: 'date'
  }

  let component = shallow(<TableMonth {...props} />);

  it('renders TableMonth component', () => {
    expect(component).toMatchSnapshot();
  })

  it('checks TableMonth date', () => {
    expect(component.find('h2').text()).toEqual('date');
  })
})
