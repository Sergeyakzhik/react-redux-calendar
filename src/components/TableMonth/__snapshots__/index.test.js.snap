// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableMonth render TableMonth component 1`] = `
<div>
  <h2>
    date
  </h2>
  <table
    className="table-month"
  >
    <thead />
    <tbody />
  </table>
</div>
`;
